code/game/machinery/computer/cloning.dm

```diff
@@ -456,6 +456,10 @@
 		scantemp = "<span class='bad'>Subject no longer contains the fundamental materials required to create a living clone.</span>"
 		playsound(src, 'sound/machines/terminal_alert.ogg', 50, 0)
 		return
+	if(dna && dna.species && (DIFFICULTCLONE in dna.species.species_traits) && (src.scanner.scan_level < 2))
+		scantemp = "<span class='bad'>Subject's DNA is too complex or damaged for current components to scan.</span>"

```

Comment:
Can this be more descriptive? "Upgrade components to scan subject" or something.